docs/source/adding_new_benchmark.rst

```diff
@@ -0,0 +1,180 @@
+Adding a New Benchmark
+######################
+
+This guide explains how to integrate a new benchmark into **metriq-gym**. You'll learn how to:
+
+1. Define the core Python structures (classes and :code:`dataclass` objects).
+2. Create a JSON Schema to validate benchmark parameters.
+3. Provide an example schema for convenience.
+4. Register the new benchmark within the package so it becomes accessible to the rest of the system.
+
+Defining a New Benchmark
+************************
+
+1. **Create a New Python File**
+
+   In the :file:`benchmarks` directory of the :code:`metriq-gym` package, add a new file named
+   :file:`benchmarks/<NEW_BENCHMARK>.py` (replace :code:`<NEW_BENCHMARK>` with a descriptive name).
+
+2. **Implement the Benchmark Class**
+
+   In this new file, define a class that inherits from :code:`Benchmark`. You must override:
+
+   - :code:`dispatch_handler()`: Houses the logic for dispatching and scheduling the benchmark job to the quantum device.
```

Comment:
```suggestion
   - :code:`dispatch_handler()`: Houses the logic for preprocessing and dispatching the benchmark job to the quantum device. This may includes things like creating the circuits, running those circuits on classical simulators, etc.
```